src/integration/python/knowrob_module.cpp

```diff
@@ -76,6 +78,38 @@ static inline void register_triple_types() {
 	boost::python::class_<TripleList>("TripleList").def(boost::python::vector_indexing_suite<TripleList, true>());
 }
 
+static void InitKnowledgeBaseWrapper(boost::python::list py_argv) {
+	static int argc = boost::python::len(py_argv);
+	std::vector<std::string> arg_strings;
```

Comment:
as explained on zulip, this must also be static.
The reason, the vector<char*> points to the memory allocated by vector<std::string>. **so both must be static**